hypha/static_src/src/sass/apply/components/_rich-text.scss

```diff
@@ -47,23 +60,12 @@
         }
     }
 
-    &__table {
-        overflow: auto;
```

Comment:
I believe we need to keep this. Some tables are really big/wide and they break the layout otherwise.

Reply:
I could not reproduce and it looks will only happen if `<table>` has an explicit width defined on it. But I agree it will break the layout and overflow into the sidebar.

I've added it back and also increased the width of the container to be more wide and take the full space.